tools/tagger/index.html

```diff
@@ -554,12 +562,26 @@
         });
       });
     }
+
+    function setLang() {
+      const lang = document.getElementById('lang-switcher');
+      const url = new URL(window.location.href);
+      url.searchParams.set('lang', lang.value);
+      window.location.href = url.toString();
+    }
   </script>
 </head>
 
 <body>
   <div id="tags-data"></div>
   <main>
+    <div class="lang">
+      <select id="lang-switcher" onchange="setLang()">
+        <option value="en">English taxonomy</option>
+        <option value="ko">Korean taxonomy</option>
+        <option value="de">German taxonomy</option>
```

Comment:
```suggestion
        <option value="en">English</option>
        <option value="ko">Korean</option>
        <option value="de">German</option>
```
I wouldn't repeat "taxonomy" in every option.

Reply:
I had that initially but since the text does not translate (it is now into the HTML), I want to make it clear that it is only the taxonomy part.